### PR TITLE
Add link to Slack

### LINKED ISSUE #15: Slack link
Hello @LindsayLF,

I wanted to add the Slack community link to this page, but the [actual link ](https://openssf.slack.com/join/shared_invite/zt-kjkth2mc-ELii08LEup2urk8Suy0Q1Q#/)from the website is not working:
Screenshot from 2021-01-29 13-24-13

Would you have a new link so it can be added here and fixed on the website?

Best,
Julien

### PR DESCRIPTION
**Pull request by jstoja**
Hi,

Slack link was missing and broken.
This PR adds the correct link in the readme.

Closes #15

### CODE REVIEW

**Comment by dlorenc** [comment]
Thanks!

**Comment by dlorenc** [comment]
Whoops! I think there's a typo here.

File: README.md [hunk 1]
```diff
@@ -21,3 +21,4 @@ Open source software has become pervasive in data centers, consumer devices, and
 * [Open SSF Community Calendar](https://calendar.google.com/calendar/r?cid=czYzdm9lZmhwNWk5cGZsdGI1cTY3bmdwZXNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
 * [Open SSF Mailing Lists](https://lists.openssf.org/g/main)
 * [Open SSF web site](https://openssf.org/)
+* [Open SSF Slack](https://slack.openssl.org/)
```
Comment (dlorenc) [inline README.md:24]: I think this should be "slack.openssf.org"

**Comment by jstoja** [comment]
Damn muscle memory! It's fixed now :)

**Pull request merged**